src/main/java/uk/gov/hmcts/reform/lrdapi/domain/Region.java

```diff
@@ -41,6 +41,9 @@
     @Column(name = "updated_time")
     private LocalDateTime updatedTime;
 
+    @Column(name = "api_enabled")
+    private boolean apiEnabled;
+
     @OneToMany(targetEntity = BuildingLocation.class, mappedBy = "region")
```

Comment:
I think we should have Fetch Mode SUBSELECT else it may end up n+1 queries not your code issue existing system code.

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-api/pull/273/commits/026e3cf0fd324a5d1408d6eb33f408f984cab036